client/setup.py

```diff
@@ -22,9 +22,10 @@
     "commonmark>=0.9.1",
     "textual==0.1.18",
     "jsonlines==3.0.0",
-    "boto3==1.21.0",
+    "boto3==1.26.22",
```

Comment:
Do we really rely on this version exactly? (while the dill pkg uses `>=`)

Reply:
ok